Makefile

```diff
@@ -158,6 +163,30 @@ TOPDIR:=$(shell pwd)
 all: deb rpm calico/felix-$(ARCH)
 test: ut fv
 
+all-multi: $(addprefix sub-all-,$(ARCHES))
+sub-all-%:
+	$(MAKE) all ARCH=$*
+
+manifest-tool:
```

Comment:
Bit of a hack to download this here.  I think we should add this to the go-build container so that it's available to all the builds at a pinned / blessed version.

Reply:
Nice idea,! instead of downloading the flow will be copying the manifest-tool from `go-build` container to host machine for building manifest image(as it requires docker daemon access, credentials to push to docker hub etc..)

Is my understanding correct?

Reply:
Yes, perhaps that's the best we can do.  We also do that for ginkgo, in order to run the FV tests is a way that lets them use docker.  Suggest copying the binary out into ./bin, which I think is what we do for ginkgo when needed.

Reply:
cool.

Reply:
I'll add a PR to `go-build` to add manifest-tool binary. -Thanks

Reply:
@fasaxc : submitted the PR to include manifest-tool in `go-build` - https://github.com/projectcalico/go-build/pull/39

Reply:
Why copy it to the host machine? Why not just run it in a container and bind-mount the docker config in?

Reply:
well, theoretically yes. But not really sure whether container will also get the docker credentials also inside the container to push images to the registry.

Reply:
Either way, once @mkumatag's PR adding it to go-build is in, I will change this PR to leverage it.

Reply:
Updated to use the `manifest-tool` in `calico/go-build:v0.14`